js/pro/timex.js

```diff
@@ -0,0 +1,622 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+const timexRest = require ('../timex');
+const { ArgumentsRequired, ExchangeError } = require ('../base/errors');
+const { ArrayCache, ArrayCacheBySymbolById } = require ('./base/Cache');
+
+//  ---------------------------------------------------------------------------
+
+module.exports = class timex extends timexRest {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'has': {
+                'ws': true,
+                'watchBalance': true,
+                'watchTicker': true,
+                'watchTickers': false,
+                'watchTrades': true,
+                'watchMyTrades': false,
+                'watchOrders': true,
+                'watchOrderBook': true,
+                'watchOHLCV': true,
```

Comment:
@pcriadoperez are we missing this implementation? this flag is true but the method does not exist

https://docs.timex.io/websocket-api-list-streams.html#candles

Reply:
It's been a while since I looked at this. But at the time I think I decided against using any of the REST messages under websockets.
I just tried it now with `/public/trades` and it will only send one response after requesting the trades instead of a constant stream. Therefore working more like a REST api call instead of websocket subscription.
So I thought it was better not to implement in this PR and use ccxt regular REST calls.

Let me know what you think, but for that I'll put `watchOHLCV: false` as there is no subscription stream for it.